src/main.c

```diff
@@ -672,13 +674,22 @@ void handleApdu(unsigned int *flags, unsigned int *tx) {
                     break;
 
                 case INS_SIGN_EIP_712_MESSAGE:
-                    memset(tmpCtx.transactionContext.tokenSet, 0, MAX_ITEMS);
-                    handleSignEIP712Message(G_io_apdu_buffer[OFFSET_P1],
-                                            G_io_apdu_buffer[OFFSET_P2],
-                                            G_io_apdu_buffer + OFFSET_CDATA,
-                                            G_io_apdu_buffer[OFFSET_LC],
-                                            flags,
-                                            tx);
+                    if (G_io_apdu_buffer[OFFSET_P2] == 0) {
```

Comment:
P2_NO_CHAINCODE ?

Reply:
Fixed in [4db1e99](https://github.com/LedgerHQ/app-ethereum/pull/327/commits/4db1e99fabbae80aa4288b035f4a012d5c6273f2)